pulp_ansible/tests/functional/constants.py

```diff
@@ -38,15 +38,17 @@
 # FIXME: replace this with the location of one specific content unit of your choosing
 ANSIBLE_URL = urljoin(ANSIBLE_FIXTURE_URL, "")
 
-ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy-dev.ansible.com"
+ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy.ansible.com/api/v2/collections/"
+
+ANSIBLE_COLLECTION_FIXTURE_URL = urljoin(ANSIBLE_GALAXY_COLLECTION_URL, NAMESPACE_ANSIBLE)
```

Comment:
this filter https://galaxy.ansible.com/api/v2/collections/?namespace__name=ansible is not working, so we have 32 pages, and sometimes tests break because it takes more than 10 minutes

Reply:
I'm confused. https://galaxy.ansible.com/api/v2/collections/?namespace__name=ansible consistently yields 32 results (not pages).

Reply:
sorry, it was results, not pages, but it is not filtering, you can see that some results have different names

Reply:
this tests could take more than 10 minutes: `test_successive_syncs_repo_version` because of it: https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/tests/functional/api/collection/test_sync.py#L87-L89

Reply:
@fabricio-aguiar oh I see what you mean. The filter results are incorrect coming back from galaxy. Can you open an issue here https://github.com/ansible/galaxy/issues and then bring it up in #pulp-ansible

Reply:
@fabricio-aguiar  For the full mirror test, I think we want a test that does that but we want to have Travis running the tests exclude that test. It would be great if the nightly cron ran it. I'm also ok for deleting the full mirror test for now and filing an issue about making this cron/PR nightly test separation separately. What do you think?